app/src/main/java/endava/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,51 @@
+package endava.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Preview
+@Composable
+fun UserScoreProgressBar(grade: Float = 10f) {
+    Box(contentAlignment = Alignment.Center)
+    {
+        val tempGrade: String = grade.toString()
+        Text(text = tempGrade, color = Color.Green, fontWeight = FontWeight.Bold, fontSize = 20.sp)
+        Canvas(
+            modifier = Modifier
+                .size(size = 100.dp)
+        ) {
+            drawArc(
```

Comment:
You don't need to change this, but just for the info: there is a `drawCircle` method which will draw the full circle, so you don't have to set start and sweep angles.